.github/workflows/data-builder-container.yml

```diff
@@ -1,15 +1,16 @@
 name: Build and push the data builder image to GitHub Container Registry
 on:
   push:
```

Comment:
Since we opened the possibility to dispatch builds manually, I wonder if we could limit automated `dev` builds to when feature branches are merged into `main`. My only concern, that is not solidly grounded, is that we might trigger many builds by pushing small fixes to `main` that do not warrant re-builds (e.g. `Update README`)

Reply:
And on the related note of avoiding unnecessary builds I make this note for later inquiry: if we make a documentation PR to main, is there a way to `mark` the particular PR so that merging it do not trigger a new image build ?

The most general version of this question would be: could we skip re-builds for PR that touches only a set of files (for example always avoid rebuilds of hugo site when only the `scripts` directory receives changes.

This is all food for thought, should we happen to have a light sprint or something.

Reply:
Great idea, [this is quite straightforward to setup which is nice](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore), and I have something similar in the lighthouse.yml gh action already. 

I've now added a setting to prevent rebuilds if only the README.md file is updated in commit 3cc2e149c64c227e84d2b972ef30c3bd7c9a22ee. This could be made better after a good think about exactly what files to commit but I think this should suffice for now.